Assets/__Scripts/MapEditor/Mapping/Strobe Generator/Passes/StrobeLaserSpeedInterpolationPass.cs

```diff
@@ -86,7 +86,8 @@ public override IEnumerable<MapEvent> StrobePassForLane(IEnumerable<MapEvent> or
             var decimalPreciseSpeed = Math.Round(Mathf.Lerp(lastSpeed, nextSpeed, easingFunc(progress)), decimalPrecision);
             MapEvent data = new MapEvent(newTime, type, 1);
             data._customData = new JSONObject();
-            data._customData["_preciseSpeed"] = decimalPreciseSpeed;
+            data._value = (int)(Math.Round(decimalPreciseSpeed, MidpointRounding.AwayFromZero) == 0 && decimalPreciseSpeed != 0 ? 1 : Math.Round(decimalPreciseSpeed, MidpointRounding.AwayFromZero));
```

Comment:
This can easily be simplified to something like

![image](https://user-images.githubusercontent.com/30004286/105612946-3911f600-5d74-11eb-89ff-8531d3588697.png)

Reply:
That will still change a decimalPreciseSpeed of 0 to 1, right? Other than that it's definitely better.

Reply:
Yes, the `Max` method would take the higher value of the two numbers. With a decimalPreciseSpeed of 0 (and below), it will return 1

Reply:
This solution doesn't handle negative numbers (see discussion below)

Reply:
tbf, i dont think we need to support negative numbers, since they'd behave the exact same as positive numbers (with the direction flipped)

Reply:
Well that's the exact question I asked.

Reply:
0 should probably stay as 0, not default to 1.
And like Caeden said, I didn't think handling negative numbers would be necessary.